app/views/disputes/strikes/index.html.haml

```diff
@@ -0,0 +1,4 @@
+- content_for :page_title do
+  = t('settings.strikes')
+
```

Comment:
```suggestion

%p= t('disputes.strikes.description_html', instance: Rails.configuration.x.local_domain)

```

"These are actions taken against your account and warnings that have been sent to you by the staff of %{instance}"